src/web/components/SlotBodyEnd.tsx

```diff
@@ -33,6 +33,12 @@ export const SlotBodyEnd = ({
     isPaidContent,
     tags,
 }: Props) => {
+    // Hardcoding the test ID until we have A/B testing (variants) capability in DCR.
+    // Currently this matches the local vs remote A/B test running on Frontend.
+    const epicTestName = 'RemoteRenderEpic';
```

Comment:
Do we want to use something different here to avoid confusion/conflating results?

Reply:
I've changed it to `DotcomRenderingEpic`